packages/wonder-blocks-popover/src/components/popover.tsx

```diff
@@ -233,12 +253,12 @@ export default class Popover extends React.Component<Props, State> {
                     {(props: PopperElementProps) => (
                         <PopoverDialog
                             {...props}
-                            aria-describedby={`${uniqueId}-anchor`}
+                            aria-describedby={`${uniqueId}-${describedBy}`}
```

Comment:
So it looks like this existing `aria-describedby` prop makes it so the popover is described by the content that it is pointing at. I think it would be best not to totally remove that, as we want some way to tie the popover to the content it points at. [Adobe's `react-aria-components` library](https://react-spectrum.adobe.com/react-aria/Popover.html) uses `aria-labelledby` for this, so maybe a solution would be:
```suggestion
                            aria-labelledby={`${uniqueId}-anchor`}
                            aria-describedby={`${uniqueId}-${describedBy}`}
```

Reply:
In this case pointing `labelledby` to the anchor element would not work as we expect. If we make that, then the dialog would be announced with the contents of the button/anchor element and the intention would be to use that to announce the dialog title instead.

From: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role#associated_aria_roles_states_and_properties

> [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby)
Use this attribute to label the dialog. Often, the value of the aria-labelledby attribute will be the id of the element used to title the dialog.

That makes me think that we can even use that instead of the `describedBy` solution. Let me try that alternative.